src/server/scripts/Northrend/Ulduar/Ulduar/boss_algalon_the_observer.cpp

```diff
@@ -1026,6 +1026,13 @@ class npc_algalon_worm_hole : public CreatureScript
             _summonTimer = urand(22000, 24000);
         }
 
+        void JustSummoned(Creature* summon) override
+        {
+            if (TempSummon* summ = me->ToTempSummon())
+                if (Creature* algalon = ObjectAccessor::GetCreature(*me, summ->GetSummonerGUID()))
+                    algalon->AI()->JustSummoned(summon);
+        }
+
```

Comment:
```suggestion
        void JustSummoned(Creature* summon) override
        {
            if (auto summ = me->ToTempSummon())
            {
                if (auto algalon = ObjectAccessor::GetCreature(*me, summ->GetSummonerGUID()))
                {
                    algalon->AI()->JustSummoned(summon);
                }
            }
        }
```

Reply:
Why auto? I don't get it.

Reply:
the `auto` is just a suggestion to save some typing, not mandatory. Please add the brackets though. It may sound too strict but I've seen many mistakes happening because of the lack of those. Including in AC.